tests/test_run_rvic.py

```diff
@@ -3,10 +3,20 @@
 from osprey_flask_app import run_rvic
 from wps_tools.testing import url_path
 from tempfile import NamedTemporaryFile
+from pkg_resources import resource_filename
 import os
 import requests
 
 
+def full_rvic_test(kwargs):
+    outpath = run_rvic.run_full_rvic(kwargs)
+    outpath_url = requests.get(outpath)
+    with NamedTemporaryFile(suffix=".nc", dir="/tmp") as outfile:
+        outfile.write(outpath_url.content)
+        assert os.path.isfile(outfile.name)
+        outfile.close()
```

Comment:
When using a context manager `.close()` is taken care of for you once the scope is left.